server.js

```diff
@@ -65,12 +65,12 @@ app.get("/", (req, res) => {
     res.render("index.ejs");
 })
 
-app.get("/future", (req, res) => {
-    res.render("future_plans.ejs");
+app.get("/plans", (req, res) => {
+    res.render("plans.ejs");
 })
 
 app.get("/start", (req, res) => {
-    res.render("questions.ejs", {questionsTextJSON: getQuestions(req.cookies.locale)});
+    res.render("questions.ejs", {questionsTextJSON: getQuestions(i18n.getLocale(req))});
```

Comment:
Why does this cause a problem? Cookies from the client side should be accessible at this point.